components/Navigation.tsx

```diff
@@ -427,6 +427,27 @@ export default function Navigation() {
         >
           Invite
         </NavigationLink>
+        <NavigationLink
+          href="/resources/poll"
+          subLinks={
+            <Fragment>
+              <NavigationSubLink href="/resources/poll#poll-object">Poll Object</NavigationSubLink>
+              <NavigationSubLink href="/resources/poll#poll-create-request-object">
+                Poll Create Request Object
+              </NavigationSubLink>
+              <NavigationSubLink href="/resources/poll#layout-type">Layout Type</NavigationSubLink>
```

Comment:
```suggestion
```
Generally, any object that is only referenced by one other object should be documented as a structure in that object and not mentioned in the navigation.